project-base/storefront/components/Layout/Footer/NewsletterForm/newsletterFormMeta.tsx

```diff
@@ -14,8 +15,8 @@ export const useNewsletterForm = (): [UseFormReturn<NewsletterFormType>, Newslet
     const { t } = useTranslation();
     const resolver = yupResolver(
         Yup.object().shape<Record<keyof { email: string; privacyPolicy: boolean }, any>>({
-            email: Yup.string().required(t('This field is required')).email(t('This value is not a valid email')),
-            privacyPolicy: Yup.bool().oneOf([true], t('You have to agree with our privacy policy')),
+            email: validateEmail(t),
+            privacyPolicy: Yup.boolean().isTrue(t('You have to agree with our privacy policy')),
```

Comment:
I can see this code in at least 3 places. Maybe it would be nice to extract it...